CONTRIBUTING.md

```diff
@@ -26,6 +26,7 @@ If you're looking for guidance on filing a bug report or getting support, please
 - [Bumping Java dependencies](#bumping-java-dependencies)
 - [Rake tasks](#rake-tasks)
 - [Making a release](#making-a-release)
+- [Fuzzing your gumbo-parser changes](#fuzzing-your-changes)
```

Comment:
I don't think this link is correct ... I'd recommend running `rake format:toc` to generate the table of contents.

Reply:
removed and added under running tests anyway